### DISCUSSION TITLE
Openplayerjs icons

### DISCUSSION THREAD

**Discussion by Dacogetul** [openplayerjs/openplayerjs#415] [General]
Hello
Can you tell me a link where I can download various icons for playing forward/backward or for the next/previous movie so that I can keep the style of the player? I want to download them so that they run from their own website so that they also work locally (without an internet connection).

**Comment by rafa8626** [comment]
@Dacogetul you can find free icons at https://remixicon.com/ or https://www.svgrepo.com/

**Reply by Dacogetul** [reply]
Thank you !
